public/Lab-Files/Lesson02/Challenges/text-level-elements.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>text-level-elements</title>
+</head>
+<body>
+    <p><strong>Web development</strong> is a broadterm for the work involved in developing a <em>web</em> <em>site</em> for the Internet (<em>World</em> <em>Wide</em> <em>Web</em>) or an intranet (a <em>private</em> <em>network</em>). <em>Web</em> <em>development</em> can range from developing the simplest static single page of plain text to the most complex <em>web-based</em> <em>internet</em> <em>applications</em>, <em>electronic</em> <em>businesses</em>, and <em>social</em> <em>network</em> <em>services</em>. A more comprehensive list of tasks to which <strong>web</strong> <strong>development</strong> commonly refers, may include <em>web</em> <em>engineering</em>, <em>web</em> <em>design</em>, <em>web</em> <em>content</em> <em>development</em>, <em>client</em> <em>liaison</em>, <em>client-side/server-side</em> <em>scripting</em>, <em>web</em> <em>server</em> and <em>network</em> <em>security</em> <em>configuration</em>, and <em>e-commerce</em> <em>development</em>. Among web professionals,<strong>"web development"</strong> usually refers to the main <em>non-design</em> aspects of building web sites: writing <em>markup</em> and <em>coding</em>. Most recently<strong> Web</strong> <strong>development</strong> has come to mean the creation of <em>content</em> <em>management</em> <em>systems</em> or <strong>CMS</strong>. These <strong>CMS</strong> can be made from scratch, <em>proprietary</em> or <em>open</em> <em>source</em> . In broad terms the CMS acts as <em>middleware</em>  between the <em>database</em>  and the user through the <em>browser</em> . A principle benefit of a <strong>CMS</strong> is that it allows <em>non-technical</em> people to make changes to their web site without having <em>technical</em> knowledge.</p>
+    </body>
+</html>
```

Comment:
Amazing job, Albert! I can tell you have been working hard. You validated all of your code and completed all of the challenges.